mentat/command/command.py

```diff
@@ -37,18 +49,24 @@ def get_command_names(cls) -> list[str]:
             if not command.hidden
         ]
 
-    @classmethod
-    def get_command_completions(cls) -> List[str]:
-        return list(map(lambda name: "/" + name, cls.get_command_names()))
-
     @abstractmethod
     async def apply(self, *args: str) -> None:
         pass
 
-    # TODO: make more robust way to specify arguments for commands
     @classmethod
     @abstractmethod
-    def argument_names(cls) -> list[str]:
+    def arguments(cls) -> List[CommandArgument]:
+        pass
```

Comment:
Since a super majority of commands have no arguments I think it makes sense to make a default implementation for these two methods that returns `[]`.

Reply:
I was thinking about this, but I think it's best to keep it explicit because otherwise I could see somebody adding a command and not realizing that they can add these functions. This way, it barely takes any extra time when adding a command, and you have to at least think about whether or not you want auto completions and arguments for this command.